templates/registration/login.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+
+  <form method="post" action="{% url 'login' %}">

```

Comment:
Ensure that the URL name 'login' used in the form action matches the name defined in your `urls.py` for the login view. This is crucial for the form to submit to the correct endpoint.